### ISSUE TITLE
Signal chrashes when I try to select a sticker

### ISSUE THREAD

**Issue by HyperCriSiS** [signalapp/Signal-Android#13194]
Hi,

Signal crashes when I want to send a sticker.

Let's begin with a checklist: Replace the empty checkboxes [ ] below with checked ones [x] accordingly. -->

- [x] I have searched open and closed issues for duplicates
- [x] I am submitting a bug report for existing functionality that does not work as intended
- [x] I have read https://github.com/signalapp/Signal-Android/wiki/Submitting-useful-bug-reports
- [x] This isn't a feature request or a discussion topic

----------------------------------------

### Bug description
Since I installed some sticker packs, signal crashes shortly after I open the sticker picker.
It also crashes when a sticker is suggested according to the smiley which I chose.

### Steps to reproduce
Open sticker picker

**Actual result:** Sticker overview opens for a brief moment. Signal closes.
**Expected result:** The sticker picker should open

### Device info
**Device:** Asus Zenfone 8
**Android version:** 11
**Signal version:** 6.34.2

### Link to debug log
The Signal internal debug upload fails.

Which alternative shall I use?

Thanks
Hyper

Here is the concerning Logcat:

```
*** *** *** *** *** *** *** *** *** *** *** *** *** *** *** ***
LineageOS Version: '18.1-20220325-NIGHTLY-sake'
Build fingerprint: 'asus/WW_I006D/ASUS_I006D:11/RKQ1.201022.002/30.12.112.72:user/release-keys'
Revision: '0'
ABI: 'arm64'
Timestamp: 2023-09-27 22:48:43+0200
pid: 16880, tid: 17204, name: glide-source-th  >>> org.thoughtcrime.securesms <<<
uid: 10455
signal 6 (SIGABRT), code -1 (SI_QUEUE), fault addr --------
Abort message: 'JNI DETECTED ERROR IN APPLICATION: buf == null
    in call to SetPrimitiveArrayRegion
    from android.graphics.Bitmap org.signal.glide.webp.WebpDecoder.nativeDecodeBitmap(byte[])'
    x0  0000000000000000  x1  0000000000004334  x2  0000000000000006  x3  0000007b41e922c0
    x4  fefefefefefefeff  x5  fefefefefefefeff  x6  fefefefefefefeff  x7  7f7f7f7f7f7f7f7f
    x8  00000000000000f0  x9  1345a7d11bff45ac  x10 0000000000000000  x11 ffffffc0fffffbdf
    x12 0000000000000001  x13 00000000000000e4  x14 ff00000000000000  x15 ffffffffffffffff
    x16 0000007e44695c80  x17 0000007e44677950  x18 0000007b11eac000  x19 00000000000041f0
    x20 0000000000004334  x21 00000000ffffffff  x22 0000000000000058  x23 0000000000000058
    x24 0000007bb292a0b7  x25 0000000000000001  x26 0000007bb294100c  x27 0000007bb2f37000
    x28 b400007bd2f61c00  x29 0000007b41e92340
    lr  0000007e4462a0ec  sp  0000007b41e922a0  pc  0000007e4462a118  pst 0000000000001000
backtrace:
      #00 pc 000000000004e118  /apex/com.android.runtime/lib64/bionic/libc.so (abort+164) (BuildId: fa8b451fd81bd3a4b793dfd0dae7726c)
      #01 pc 0000000000550658  /apex/com.android.art/lib64/libart.so (art::Runtime::Abort(char const*)+2256) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #02 pc 0000000000013978  /system/lib64/libbase.so (android::base::SetAborter(std::__1::function<void (char const*)>&&)::$_3::__invoke(char const*)+76) (BuildId: 5ddeb405cbf5aa6e6ee96e6f09211cdd)
      #03 pc 0000000000012fa4  /system/lib64/libbase.so (android::base::LogMessage::~LogMessage()+320) (BuildId: 5ddeb405cbf5aa6e6ee96e6f09211cdd)
      #04 pc 0000000000381488  /apex/com.android.art/lib64/libart.so (art::JavaVMExt::JniAbort(char const*, char const*)+2548) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #05 pc 00000000003eb728  /apex/com.android.art/lib64/libart.so (art::JNI<false>::SetIntArrayRegion(_JNIEnv*, _jintArray*, int, int, int const*)+1884) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #06 pc 000000000001638c  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/lib/arm64/libsignalwebp.so (BuildId: c25417943f8d1e5cc173629c80c65c00e7a1e366)
      #07 pc 00000000000166a8  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/lib/arm64/libsignalwebp.so (BuildId: c25417943f8d1e5cc173629c80c65c00e7a1e366)
      #08 pc 00000000002529f8  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (art_jni_trampoline+152)
      #09 pc 0000000000133564  /apex/com.android.art/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #10 pc 00000000001a8534  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+200) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #11 pc 00000000003155b4  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+376) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #12 pc 000000000030bb28  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+996) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #13 pc 0000000000673da4  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualQuick+668) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #14 pc 0000000000131594  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_quick+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #15 pc 00000000015b3f90  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (org.signal.glide.webp.WebpInputStreamResourceDecoder.decode+72)
      #16 pc 0000000000674044  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualQuick+1340) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #17 pc 0000000000131594  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_quick+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #18 pc 00000000015b3fca  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (org.signal.glide.webp.WebpInputStreamResourceDecoder.decode+4)
      #19 pc 000000000066f070  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+1800) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #20 pc 000000000012da14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #21 pc 0000000000fb5490  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.DecodePath.decodeResourceWithList+64)
      #22 pc 0000000000672fac  /apex/com.android.art/lib64/libart.so (MterpInvokeDirectRange+844) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #23 pc 000000000012dc14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct_range+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #24 pc 0000000000fb541c  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.DecodePath.decodeResource+36)
      #25 pc 000000000066fcac  /apex/com.android.art/lib64/libart.so (MterpInvokeDirect+1240) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #26 pc 000000000012d914  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #27 pc 0000000000fb53d6  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.DecodePath.decode)
      #28 pc 0000000000674788  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualQuickRange+944) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #29 pc 0000000000131614  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_range_quick+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #30 pc 0000000000fb63fe  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.LoadPath.loadWithExceptionList+62)
      #31 pc 0000000000672fac  /apex/com.android.art/lib64/libart.so (MterpInvokeDirectRange+844) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #32 pc 000000000012dc14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct_range+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #33 pc 0000000000fb638e  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.LoadPath.load+38)
      #34 pc 0000000000674788  /apex/com.android.art/lib64/libart.so (MterpInvokeVirtualQuickRange+944) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #35 pc 0000000000131614  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_virtual_range_quick+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #36 pc 0000000000fb5134  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.DecodeJob.runLoadPath+50)
      #37 pc 000000000066fcac  /apex/com.android.art/lib64/libart.so (MterpInvokeDirect+1240) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #38 pc 000000000012d914  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_direct+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #39 pc 0000000000fb4f88  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.engine.DecodeJob.decodeFromFetcher+20)
      #40 pc 000000000030326c  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.811804197400490633)+268) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #41 pc 000000000065c298  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+780) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #42 pc 000000000013cff8  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #43 pc 0000000000778644  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.decodeFromData+180)
      #44 pc 00000000007789d4  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.decodeFromRetrievedData+516)
      #45 pc 000000000077a0d4  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.onDataFetcherReady+404)
      #46 pc 000000000077e6f4  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.SourceGenerator.onDataReadyInternal+356)
      #47 pc 000000000077e35c  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.SourceGenerator$1.onDataReady+140)
      #48 pc 0000000000133564  /apex/com.android.art/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #49 pc 00000000001a8534  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+200) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #50 pc 00000000003155b4  /apex/com.android.art/lib64/libart.so (art::interpreter::ArtInterpreterToCompiledCodeBridge(art::Thread*, art::ArtMethod*, art::ShadowFrame*, unsigned short, art::JValue*)+376) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #51 pc 000000000030bb28  /apex/com.android.art/lib64/libart.so (bool art::interpreter::DoCall<false, false>(art::ArtMethod*, art::Thread*, art::ShadowFrame&, art::Instruction const*, unsigned short, art::JValue*)+996) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #52 pc 000000000066ed68  /apex/com.android.art/lib64/libart.so (MterpInvokeInterface+1024) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #53 pc 000000000012da14  /apex/com.android.art/lib64/libart.so (mterp_op_invoke_interface+20) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #54 pc 0000000000fb42ca  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.vdex (com.bumptech.glide.load.data.LocalUriFetcher.loadData+20)
      #55 pc 000000000030326c  /apex/com.android.art/lib64/libart.so (art::interpreter::Execute(art::Thread*, art::CodeItemDataAccessor const&, art::ShadowFrame&, art::JValue, bool, bool) (.llvm.811804197400490633)+268) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #56 pc 000000000065c298  /apex/com.android.art/lib64/libart.so (artQuickToInterpreterBridge+780) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #57 pc 000000000013cff8  /apex/com.android.art/lib64/libart.so (art_quick_to_interpreter_bridge+88) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #58 pc 000000000077e4e8  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.SourceGenerator.startNextLoad+200)
      #59 pc 000000000077ea80  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.SourceGenerator.startNext+848)
      #60 pc 000000000077969c  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.runGenerators+156)
      #61 pc 0000000000779a64  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.runWrapped+404)
      #62 pc 000000000077a298  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.DecodeJob.run+184)
      #63 pc 00000000002a6f54  /apex/com.android.art/javalib/arm64/boot.oat (java.util.concurrent.ThreadPoolExecutor.runWorker+980) (BuildId: fd3cf054c28fb5d3737a5d92c072cfb27c32c982)
      #64 pc 00000000002a4150  /apex/com.android.art/javalib/arm64/boot.oat (java.util.concurrent.ThreadPoolExecutor$Worker.run+64) (BuildId: fd3cf054c28fb5d3737a5d92c072cfb27c32c982)
      #65 pc 00000000006caecc  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.executor.GlideExecutor$DefaultThreadFactory$1.run+236)
      #66 pc 000000000015dd08  /apex/com.android.art/javalib/arm64/boot.oat (java.lang.Thread.run+72) (BuildId: fd3cf054c28fb5d3737a5d92c072cfb27c32c982)
      #67 pc 00000000006cacd0  /data/app/~~fUlDJwDBPOKERs2J96MpBA==/org.thoughtcrime.securesms-wREvsLACSrXFYdoWQsKWpw==/oat/arm64/base.odex (com.bumptech.glide.load.engine.executor.GlideExecutor$DefaultPriorityThreadFactory$1.run+64)
      #68 pc 0000000000133564  /apex/com.android.art/lib64/libart.so (art_quick_invoke_stub+548) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #69 pc 00000000001a8534  /apex/com.android.art/lib64/libart.so (art::ArtMethod::Invoke(art::Thread*, unsigned int*, unsigned int, art::JValue*, char const*)+200) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #70 pc 000000000054837c  /apex/com.android.art/lib64/libart.so (art::JValue art::InvokeVirtualOrInterfaceWithJValues<art::ArtMethod*>(art::ScopedObjectAccessAlreadyRunnable const&, _jobject*, art::ArtMethod*, jvalue const*)+460) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #71 pc 0000000000596860  /apex/com.android.art/lib64/libart.so (art::Thread::CreateCallback(void*)+1288) (BuildId: ad3417ed374ad9e2f9f7b4cf9b0fc723)
      #72 pc 00000000000b0db8  /apex/com.android.runtime/lib64/bionic/libc.so (__pthread_start(void*)+64) (BuildId: fa8b451fd81bd3a4b793dfd0dae7726c)
      #73 pc 000000000005004c  /apex/com.android.runtime/lib64/bionic/libc.so (__start_thread+64) (BuildId: fa8b451fd81bd3a4b793dfd0dae7726c)
```

**Comment by cody-signal** [comment]
Can you try updating to the latest beta, this should be fixed/not crash in 6.34.3 or later.

**Comment by HyperCriSiS** [comment]
Thank you, it is working, was not aware of the new update!
